### PR TITLE
Change to use Fluent masthead

### PR DESCRIPTION
**Pull request by vishwam**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "@microsoft/azure-iot-ux-fluent-controls": "^6.0.0-uialignment.1",
-    "@microsoft/azure-iot-ux-fluent-css": "^6.0.0-uialignment.1",
+    "@microsoft/azure-iot-ux-fluent-controls": "^6.0.0-uialignment.6",
```
Comment (vincenzocaruso) [inline package.json]: ^6.0.0-uialignment.7 to get the latest

File: src/components/shared/contextMenu/contextMenu.scss [hunk 2]
```diff
@@ -27,6 +27,11 @@ $headerHeight: 48px;
     @include rem-fallback(padding, 0px, 10px);
   }
 
+  // buttons in the context menu should be centered vertically:
```
Comment (vincenzocaruso) [inline src/components/shared/contextMenu/contextMenu.scss:30]: can you give me more context?

File: src/components/shell/shell.js [hunk 3]
```diff
@@ -86,6 +83,49 @@ class Shell extends Component {
...
 
+  getMastheadProps() {
+    const { pagesConfig, t, denyAccess, openSystemSettings, openUserProfile, openHelpFlyout, openFlyout } = this.props;
+    if (denyAccess) {
+      return {
+        branding: t('header.appName'),
+        more: {
+          icon: 'more',
+          label: t('header.more'),
+          selected: false,
```
Comment (vincenzocaruso) [inline src/components/shell/shell.js]: should this be driven by some state?

File: src/components/shared/contextMenu/contextMenu.scss (hunk 2 above)
Comment (vishwam) [reply src/components/shared/contextMenu/contextMenu.scss:30]: This is to make the buttons in the context panel align vertically

File: src/components/shell/shell.js (hunk 3 above)
Comment (vishwam) [reply src/components/shell/shell.js]: added

**Review by vincenzocaruso — APPROVED** [review]

**Pull request merged**